it/RetrieveAllLegalUnitsForEnterpriseAcceptanceSpec.scala

```diff
@@ -75,16 +77,15 @@ class RetrieveAllLegalUnitsForEnterpriseAcceptanceSpec extends ServerAcceptanceS
       response.status shouldBe OK
       response.header(CONTENT_TYPE).value shouldBe JSON
       response.json.as[Seq[LegalUnit]] should contain theSameElementsAs Seq(
-        LegalUnit(UBRNOne, crn = Some("one-crn"), name = "one-name", legalStatus = "one-legalStatus",
-          tradingStatus = "one-tradingStatus", tradingstyle = None, sic07 = "one-sic07", turnover = None,
-          jobs = Some(42), enterprise = EnterpriseLink(TargetErn, entref = None),
-          address = Address(line1 = "one-address1", line2 = Some("one-address2"), line3 = Some("one-address3"),
-            line4 = Some("one-address4"), line5 = None, postcode = "one-postcode")),
-        LegalUnit(UBRNTwo, crn = None, name = "two-name", legalStatus = "two-legalStatus",
-          tradingStatus = "two-tradingStatus", tradingstyle = Some("two-tradingstyle"),
-          sic07 = "two-sic07", turnover = Some(10), jobs = None, enterprise = EnterpriseLink(TargetErn, entref = Some("two-entref")),
-          address = Address(line1 = "two-address1", line2 = Some("two-address2"), line3 = None,
-            line4 = None, line5 = Some("two-address5"), postcode = "two-postcode"))
+        LegalUnit(ubrn = UbrnOne, name = "Big Box Cereal Ltd", tradingStyle = Some("The Cereal Co"), sic07 = "10612",
```

Comment:
It might be better to store these LEU's in a sample data trait, although this way may be better for an acceptance test as it is more explicit.

Reply:
We do have a bit of inconsistency in approach here.  The Enterprise Acceptance Spec does what you suggest and defines all of the values in a SampleEnterpriseTrait, which seems to work reasonably well.

As I am only changing a few fields here, I didn't want to make wholesale changes.